cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -512,7 +512,7 @@ def test_vm_power_options_from_off(provider, soft_assert, testing_vm, ensure_vm_
 
 
 @pytest.mark.provider([VMwareProvider, RHEVMProvider], override=True, scope='function')
-@pytest.mark.meta(blockers=[BZ(1571830, forced_streams=['5.8', '5.9'],
+@pytest.mark.meta(blockers=[BZ(1571830, forced_streams=['5.8', '5.9', '5.10'],
```

Comment:
**Required** Remove 5.8 from the `forced_streams`, we are no longer testing 5.8. Any 5.8 updates can be pushed to `5.8-breakpoint-patched`

Reply:
Thanks @jawatts :)